File: python/bubble_sort.py

N = 5
aux = -1
vetor = [None] * N

# vetor = list(range(N))

for i in range(N):
	vetor[i] = int(input('Entre com o ' + str(i + 1) + 'º valor: '))

for i in range(N):
	for j in range((N - 1)):
		if vetor[j] > vetor[j + 1]:
			aux = vetor[j]
			vetor[j] = vetor[j + 1]
			vetor[j + 1] = aux

for i in range(N):
	print(str(i + 1) + '° número: ' + str(vetor[i]), end='\n')

File: C++/bubble_sort.cpp
#include<iostream>

using namespace std;

int main() {
	int vetor[5], aux, N = 5;

	for (int i = 0; i < N; i++) {
		cout << "Digite o " << i + 1 << "° número: ";
		cin >> vetor[i];
	}

	for (int i = 0; i < N; i++) {
		for (int j = 0; j < (N - 1); j++) {
			if (vetor[j] > vetor[j + 1]) {
				aux = vetor[j];
				vetor[j] = vetor[j + 1];
				vetor[j + 1] = aux;
			}
		}
	}

	for (int i = 0; i < N; i++) {
		cout << (i + 1) << "° número: " << vetor[i] << "\n";
	}

	return 0;
}
